### ISSUE TITLE
experience performance issues

### ISSUE THREAD

**Issue by hejun041** [folofse/androw#20]
This is a very well implemented library that perfectly restores the shadow on Android

But when used in large quantities on a project, there are serious performance problems :(
is there any solutions or idea?

**Comment by hejun041** [comment]
https://developer.android.com/reference/com/google/android/material/shape/MaterialShapeDrawable

**Comment by folofse** [comment]
Yes it uses Bitmap Rendering to simulate the shadows which can be performance heavy if multiple shadows and animations are rendered at the same time. 

I have not found any other solution for Android so far that can apply to any component without faking the shadow with Bitmap Render.
